lib/main.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_projects/routes/movie_list.dart';
+import 'routes/movie_main_widget.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: MaterialApp(
+        theme: ThemeData.dark(),
+        initialRoute: '/listPreview',
+        routes: {
+          '/listPreview': (context) => MovieList(),
+          '/MovieView': (context) => const MovieMainWidget(),
```

Comment:
Use constants for the routes. You could create constants inside the widgets itself, or create an AppRouter with the routes inside.

Reply:
Still missing :(

Reply:
Now I see what you mean jajaja, addressed